otterdog/eclipse-tractusx.jsonnet

```diff
@@ -414,6 +414,20 @@ orgs.newOrg('eclipse-tractusx') {
         },
       ],
     },
+    orgs.newRepo('puris') {
+      allow_update_branch: false,
+      delete_branch_on_merge: true,
+      description: "Predictive Unit Real-Time Information Service (PURIS) for Short Term Demand and Capacity Management",
+      secret_scanning_push_protection: "disabled",
+      web_commit_signoff_required: false,
+      branch_protection_rules: [
+        orgs.newBranchProtectionRule('main') {
+          required_approving_review_count: 1,
+          requires_status_checks: false,
+          requires_strict_status_checks: true,
```

Comment:
this setting is redundant, as it would only be taken into account when requires_status_checks would be enabled.

Also, if you would enable status checks, you could enforce the eca validation checks which is defined in the default configuration, which we suggest for projects using otterdog and setting up branch protection rules. Right now the eca checks are not enforced, but when you have a branch protection rule in place you can enable it via required status checks.

Reply:
Sound good. I just updated the configuration to enable the `requires_status_checks`.

For the initial version, I used the config from the other repos because a lot of them are using these three lines. Maybe they should be updated accordingly?

Further, it would be great if the [OtterDog documentation](https://gitlab.eclipse.org/eclipsefdn/security/otterdog) could be updated because both `requires_status_check` and `requires_strict_status_check` are currently described with "TBD" which makes it hard to guess the functionality behind it.

Reply:
Great, we imported the current status quo , that's why the configuration is like that, but should be improved over time to match your needs. We dont want to impose anything, so feel free to update when you have some time.

The documentation needs to be improved indeed, I am working on a separate site at otterdog.readthedocs.io.